crates/api_crud/src/post/create.rs

```diff
@@ -50,17 +56,21 @@ pub async fn create_post(
 ) -> Result<Json<PostResponse>, LemmyError> {
   let local_site = LocalSite::read(&mut context.pool()).await?;
 
+  honeypot_check(&data.honeypot)?;
+
   let slur_regex = local_site_to_slur_regex(&local_site);
   check_slurs(&data.name, &slur_regex)?;
+
   let body = process_markdown_opt(&data.body, &slur_regex, &context).await?;
-  honeypot_check(&data.honeypot)?;
+  let alt_text = &data.alt_text;
```

Comment:
I dont think this is good code style, it just adds an extra line for zero benefit.